kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md

```diff
@@ -1,6 +1,10 @@
 % Helm
 % Lucas Bickel
 
+![](static/adfinis_sygroup_logo.png)
+
+Be smart. Think open source.
+
 # ![](static/helm_logo_transparent.png)
```

Comment:
```suggestion
```

Reply:
Why? The logo should be in the training somewhere. It serves as an important visual identifier.

Reply:
but its commented out?

Reply:
It's not a comment, it's a h1 markdown tag that forces the logo to be on it's own page.

Reply:
oops